packages/output/src/common/output-channel.ts

```diff
@@ -105,12 +105,20 @@ export class OutputChannelManager implements Disposable {
     }
 }
 
+export enum OutputChannelSeverity {
+    info, Warning, Error
```

Comment:
`info` -> `Info`?

Reply:
`packages/core/src/common/severity.ts` defines another `Severity`; can we kind of follow its enum order.
```ts
export enum Severity {
    Ignore = 0,
    Error = 1,
    Warning = 2,
    Info = 3,
    Log = 4
}
``` 
So the new `OutputChannelSeverity` would become:
```ts
export enum OutputChannelSeverity {
    Error = 1,
    Warning = 2,
    Info = 3
}
```

Reply:
fixed both